src/Metadata/Property/Factory/InheritedPropertyMetadataFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
You're right ofc, I'm adding these asap, my slow computer is upgrading atm.
On Sat, 27 Aug 2016 at 15:25, Kévin Dunglas <notifications@github.com>
wrote:

> In src/Metadata/Property/Factory/InheritedPropertyMetadataFactory.php
> <https://github.com/api-platform/core/pull/688#discussion_r76516117>:
>
> > @@ -0,0 +1,54 @@
> > +<?php
>
> Why try to achieve a 100% coverage in unit tests to be able to use each
> library (metadata, Swagger, Hydra...) as separate components. Btw unit test
> allows to better think the public API of classes than functional tests.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/api-platform/core/pull/688/files/901069559f49c8fff19255419e1c0bfec30f2108#r76516117>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQr80HJwwAm8Ht-v6SmvdAP16YibcCPks5qkDrDgaJpZM4JpsBR>
> .
>